docs/getting_started.md

```diff
@@ -129,6 +129,12 @@ To actually use it, you will need to configure some linked channels, see
 Once a Slack Workspace is connected, you can offer automatic hints on how
 to bridge existing and new channels by enabling [Workspace Sync](team_sync.md).
 
+## Upgrading
+1. Build the latest version of the application service. [Follow the Installation section instructions.](##Installation)
```

Comment:
Is `##Installation` some ReadTheDocs / Markdown syntax I don't know about to reference headings?
The HTML id of that heading is `installation`.

```suggestion
1. Build the latest version of the application service. [Follow the Installation section instructions.](#installation)
```